packages/playground/src/utils/nodeSelector.ts

```diff
@@ -229,10 +229,7 @@ export async function validateRentContract(
         id: node.rentContractId,
       });
       if (contractInfo.state.gracePeriod) {
-        createCustomToast(
-          `You can't deploy on node ${node.nodeId}, its rent contract is in grace period.`,
-          ToastType.danger,
-        );
+        throw `You can't deploy on node ${node.nodeId}, its rent contract is in grace period.`;
```

Comment:
i don't think we should throw an error here, we should update the field message. 
![image](https://github.com/user-attachments/assets/f0fb6fa5-0a8e-4e64-ad99-ddfaaafe3bdc)

Reply:
![image](https://github.com/user-attachments/assets/ca1d135d-1c3f-474b-b452-06d7648d74d9)

Reply:
I don't think so as it's doing the double toast issue and it all should take the same behavior (throwing an error or firing a toast)

Reply:
Sorry, I disagree. I don't see any validation error in the manual selector that appears on toast, all errors are shown in the field info, As you said, we need to make the behavior consistent. @AhmedHanafy725  what do you think?

Reply:
It should be consistent. The error shouldn't be in the toast, it should be a field error